fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsBuilder.java

```diff
@@ -60,6 +61,10 @@ public StatisticsBuilder putColumnStatistics(Expression expression, ColumnStatis
         return this;
     }
 
+    public Set<Map.Entry<Expression, ColumnStatistic>> getExpressionColumnStatsEntries() {
+        return expressionToColumnStats.entrySet();
```

Comment:
Directly returning a map is more general.

Reply:
We want to guarantee that every entry is add by API StatististicBuilder.put(expr, columnStats). If the map is exposed, the  guarantee may be broken.